packages/headless/src/controllers/core/facets/facet/headless-core-facet.test.ts

```diff
@@ -141,6 +142,26 @@ describe('facet', () => {
       );
     });
 
+    it('dispatches #updateFacetOptions with #freezeFacetOrder true', () => {
+      const facetValue = buildMockFacetValue({value: 'TED'});
+      facet.toggleExclude(facetValue);
+
+      expect(engine.actions).toContainEqual(
+        updateFacetOptions({freezeFacetOrder: true})
```

Comment:
Is there a converse test (that already exists)?
`updateFacetOptions()` always seems to be called with `{freezeFacetOrder: true}`?

Reply:
Well I can see that there are tests for `updateFacetOptions` in `facet-options-slice.test.ts` if that's what you mean? 

The purpose of this describe is to ensure that the correct actions are dispatched with the expected parameters when calling certain methods on the controller. The specific actions have their own tests elsewhere in the codebase to ensure the state is being updated accordingly, so I believe this should be fairly complete coverage.

Perhaps I'm misunderstanding what you're suggesting though?

Reply:
I didn't see `updateFacetOptions({freezeFacetOrder: false})` used anywhere - hence wondering why the option is always true..

Reply:
Oh gotcha! I misunderstood your comment completely 🤦 

Ya it's awfully strange. I'll provide a default value for the input parameter of `updateFacetOptions` such that we assume `freezeFacetOrder: true` unless otherwise specified 🙂